app/src/main/java/com/github/swent/echo/compose/components/Pager.kt

```diff
@@ -33,8 +35,8 @@ import kotlinx.coroutines.launch
  */
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
-fun Pager(content: List<Pair<String, @Composable () -> Unit>>) {
-    val pagerState = rememberPagerState(pageCount = { content.size })
+fun Pager(content: List<Pair<String, @Composable () -> Unit>>, initialPage: Int? = 0) {
```

Comment:
You could use `initialPage: Int = 0` as you never need initial page to be null.

Reply:
I thought the ? was the way to allow a function to not take argument, I didn't know that we could do it this way ! Thanks for highlighting this ;)